pkg/util/util.go

```diff
@@ -69,3 +69,16 @@ func FetchURLWithRetries(url string, attempts int, duration time.Duration) ([]by
 
 	return data, err
 }
+
+// MergeMaps will merge the given maps, but it does not check for conflicts.
+// In case of conflicting keys, the map that is provided later wins.
```

Comment:
nit: how about `In case of conflicting keys, the map that is provided later overrides the previous one` ?